website/docs/guides/customize-schema-alias.md

```diff
@@ -0,0 +1,490 @@
+---
+title: Customize dbt models database, schema and alias
+id: customize-schema-alias
+description: "Learn how to properly adjust your generate_schema_name() and generate_alias_name() macros."
+displayText: Learn how to adjust your generate schema name and generate alias name.
+hoverSnippet: Learn how to adjust your generate schema name and generate alias name.
+# time_to_complete: '30 minutes' commenting out until we test
+icon: 'guides'
+hide_table_of_contents: true
+level: 'Advanced'
+recently_updated: true
+---
+
+<div style={{maxWidth: '900px'}}>
+
+## Introduction
+This guide explains how to customize the [schema](/docs/build/custom-schemas), [database](/docs/build/custom-databases), and [alias](/docs/build/custom-aliases) naming conventions in dbt to fit your data warehouse governance and design needs.
+When we develop dbt models and perform a `dbt run` or `dbt build` command, objects get created in the data warehouse.
+
+In the rest of the article, for consistency, when we refer to database, we'll be referring to a Google Cloud Platform  project or to a Databricks catalog.
+:::info A word on naming
+
+Different warehouses have different names for _logical databases_. The information in this document covers "databases" on Snowflake, Redshift, and Postgres; "projects" on BigQuery; and "catalogs" on Databricks Unity Catalog.
+
+:::
+Out-of-the-box, the default behavior of dbt is the following:
+
+- The database where the object is created will be the database configured at the [environment level in dbt Cloud](/docs/dbt-cloud-environments), or in the `profiles.yml` in dbt Core.
+
+- The schema depends on whether a [custom schema](/docs/build/custom-schemas) has been defined for the model:
+    - If no custom schema is defined, the object will be created in the schema configured in dbt Cloud (`dbt_username` for dev and the default schema for deployment environments) or the schema in the `profiles.yml` in dbt Core.
+ 
+
+- Note that for automated CI jobs, the schema name is derived from the job number and PR number: `dbt_cloud_pr_<job_id>_<pr_id>`
+- If a custom schema has been defined, it will concatenate the schema from above with the custom one.
+- For example, if the configured schema is `dbt_myschema` and the custom one is `marketing`, the objects will be created under `dbt_myschema_marketing`.
```

Comment:
wondering if this 'for example' should follow the previous bullet since theyr'e tied together? or be a sub bullet under it:

- If a custom schema has been defined, it will concatenate the schema from above with the custom one.
  - For example, if the configured schema is dbt_myschema and the custom one is marketing, the objects will be created under dbt_myschema_marketing.